cms/models.go

```diff
@@ -58,11 +58,12 @@ type Builder struct {
 }
 
 type RunTemplate struct {
-	ZebedeeRoot         string
-	EnableDatasetImport bool
-	DatasetAPIURL       string
-	DatasetAPIAuthToken string
-	ServiceAuthToken    string
+	ZebedeeRoot              string
```

Comment:
PR looks great, the only thing I can really see that might be improved on is maybe putting these in alphabetical order.